spec/controllers/api/users_controller_spec.rb

```diff
@@ -172,10 +257,183 @@
           expect(response).to have_http_status(:unauthorized)
         end
         
-        it "The user does not exist" do
+        it "The user cannot modify other users" do
           expect(@json_response['message']).to eq("You cannot delete other users")
         end
       end
     end
   end
-end
\ No newline at end of file
+
+  describe "Photo path" do
+    describe "PUT #add_photo_path" do
+      context "when valid" do
+        before do
+          request.headers["AUTHORIZATION"] = "Bearer #{@token}"
```

Comment:
Since this is shared by all tests, you can put it under a more general `describe` block. Same with `@json_response = JSON.parse(response.body)`